cohort-engine-api/src/main/java/com/ibm/cohort/engine/api/service/FHIRRestUtils.java

```diff
@@ -168,7 +174,12 @@ public static IGenericClient getFHIRClient(String fhirEndpoint, String bearerTok
 		RestFhirMeasureResolutionProvider msp = new RestFhirMeasureResolutionProvider(measureClient);
 		Measure measure = msp.resolveMeasureById(measureId);
```

Comment:
I think we'll need to add some null checks and some exception handling here. Looks like there were some changes to the RestFhirMeasureResolutionProvider since I originally wrote this code which removed some exceptions and instead returns null if a measure doesn't exist. This will cause NPE errors.

Reply:
That's a good point.
I wasn't involved in drafting the initial contract.
What should the expected response be in that case?
A `400` response code for a bad request or just an empty `MeasureParameterInfoList`?